src/shared_utilities/drivers.jl

```diff
@@ -1642,6 +1688,194 @@ function prescribed_forcing_era5(
     return (; atmos, radiation)
 end
 
+"""
+     prescribed_perturbed_forcing_era5(era5_ncdata_path,
+                             surface_space,
+                             start_date,
+                             earth_param_set,
+                             ΔT,
+                             FT;
+                             gustiness=1,
+                             max_wind_speed = nothing,
+                             c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+                             time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
+                             regridder_type = :InterpolationsRegridder,
+                             interpolation_method = Interpolations.Constant(),)
+
+A helper function which constructs the `PrescribedAtmosphere` and `PrescribedRadiativeFluxes`
+from a file path pointing to the ERA5 data in a netcdf file, the surface_space, the start date,
+and the earth_param_set, applying a change in the instantaneous temperature at each point
+of ΔT.
+
```

Comment:
Probably good to make clear that relative humidity is kept fixed in the process.